src/gh/diffCheck/diffCheck/df_geometries.py

```diff
@@ -236,11 +236,30 @@ def to_brep(self):
         """
         brep = rg.Brep()
         for face in self.faces:
-            brep.Append(face.to_brep_face())
+            brep.Append(face.to_brep_face().ToBrep())
```

Comment:
indeed. This might break the code elsewhere but my bad

Reply:
okay! took out .ToBrep()

It didn't break anything on my side :)